subprojects/shipkit/src/test/groovy/org/shipkit/internal/comparison/artifact/BintrayDefaultArtifactUrlResolverTest.groovy

```diff
@@ -9,12 +9,20 @@ class BintrayDefaultArtifactUrlResolverTest extends Specification {
 
     BintrayDefaultArtifactUrlResolver underTest
     Project project = Mock(Project)
+    ExtensionContainer extensionContainer = Mock(ExtensionContainer)
```

Comment:
ExtensionContainer is only used in the scope of setup method, let's make it a local variable there.